src/frontend/src/lib/components/manage/ManageTokens.svelte

```diff
@@ -235,37 +234,39 @@
 		>
 	</button>
 {:else}
-	<div class="container overflow-y-auto overscroll-contain pr-2 pt-1 md:max-h-[26rem]">
-		{#each tokens as token (`${token.network.id.description}-${token.id.description}`)}
-			<Card>
-				<TokenName data={token} />
-
-				<TokenLogo slot="icon" color="white" data={token} badge={{ type: 'network' }} />
-
-				<span class="break-all" slot="description">
-					{token.symbol}
-				</span>
-
-				<svelte:fragment slot="action">
-					{#if icTokenIcrcCustomToken(token)}
-						<IcManageTokenToggle {token} on:icToken={onToggle} />
-					{:else if icTokenEthereumUserToken(token)}
-						<ManageTokenToggle {token} on:icShowOrHideToken={onToggle} />
-					{:else if isBitcoinToken(token)}
-						<BtcManageTokenToggle />
-					{/if}
-				</svelte:fragment>
-			</Card>
-		{/each}
+	<div class="tokens flex flex-col overflow-y-hidden md:max-h-[26rem]">
+		<div class="tokens-scroll my-3 overflow-y-auto overscroll-contain pl-4 pr-2 pt-1">
+			{#each tokens as token (`${token.network.id.description}-${token.id.description}`)}
+				<Card>
+					<TokenName data={token} />
+
+					<TokenLogo slot="icon" color="white" data={token} badge={{ type: 'network' }} />
+
+					<span class="break-all" slot="description">
+						{token.symbol}
+					</span>
+
+					<svelte:fragment slot="action">
+						{#if icTokenIcrcCustomToken(token)}
+							<IcManageTokenToggle {token} on:icToken={onToggle} />
+						{:else if icTokenEthereumUserToken(token)}
+							<ManageTokenToggle {token} on:icShowOrHideToken={onToggle} />
+						{:else if isBitcoinToken(token)}
+							<BtcManageTokenToggle />
+						{/if}
+					</svelte:fragment>
+				</Card>
+			{/each}
+		</div>
 	</div>
 
-	<Hr />
-
 	<button
-		class="flex w-full justify-center pb-5 pt-4 text-center font-bold text-blue-ribbon no-underline"
+		class="mb-4 flex w-full justify-center pt-4 text-center font-bold text-blue-ribbon no-underline"
 		on:click={() => dispatch('icAddToken')}>+ {$i18n.tokens.manage.text.do_not_see_import}</button
 	>
+{/if}
 
+{#if !noTokensMatch}
```

Comment:
out of curiosity: why move it to its own IF condition?

Reply:
Good catch, thanks! That's a leftover of one of the 20th iterations. I removed the additional if. Code similar as main.